portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -48,10 +48,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.sendRedirect("/community.html");
   }
 
-  private String getUsername(HttpServletRequest request) {
-    return request.getParameter("username");
-  }
-
   private String getComment(HttpServletRequest request) {
     return request.getParameter("comment");
```

Comment:
For these hardcoded String literals representing parameter names and entity names, if you're using them multiple times, it's best to create constant variables for them.  Something like:
private static final String COMMENT_ENTITY = "Comment";
private static final String COMMENT_PARAMETER = "comment";
private static final String USERNAME_PARAMETER = "username";
// etc.

Alternatively, since it looks like you are using these strings in multiple files, you could define these constants in a common location, maybe as non-private static constants in the Comment class itself?

Reply:
Also, PARAMETER might not be the best.. maybe FIELD?

Reply:
I made these variables final because they never change!  However a few of them require methods that require method specific variables such as request for COMMENT_TEXT_FIELD so I did not make them global!

Reply:
Made a key class